### PR TITLE
Show the (organising) author group name on activities

### PR DESCRIPTION
**Pull request by jasperweyne**
This PR adds the name of the group that's the author of an activity in the frontend.

### CODE REVIEW

File: templates/activity/show.html.twig [hunk 1]
```diff
@@ -20,10 +20,11 @@
             <span>{{ activity.start|format_datetime('medium', 'short') }} - {{ activity.end|format_datetime('medium', 'short') }}</span>
         {% endif %}
         <span>{{ activity.location.address }}</span>
+        {% if activity.author %}
```
Comment (A-Daneel) [inline templates/activity/show.html.twig:23]: I can't see a situation where it is desirable to have an activity without an author.
Curious what your thought process was.

Comment (jasperweyne) [reply templates/activity/show.html.twig:23]: One can use Kiwi without utilizing the group system (eg. only admins), this check reflects that possibility. That's something we could change, but I feel that would be outside of the scope of this PR.

Comment (A-Daneel) [reply templates/activity/show.html.twig:23]: hadn't even considered that, my bad.
I do agree that such changes would be outside the scope of this 3 line PR.

I'll merge

**Review by A-Daneel — APPROVED** [review]
LGTM!

**Pull request merged**